src/messaging.js

```diff
@@ -68,6 +57,11 @@ export default class Messaging {
     }
   }
 
+  requestModuleClientList() {
+    console.log('Requesting client list');
```

Comment:
@jungeunyoon I see a bunch of logging to console across all the source code of the component. Curious, could some of this be reduced to not pollute the console? Are some of these more appropriate to log to BQ?

Reply:
@stulees Sure, i'll take a look and modify. Would make troubleshooting easier since then we wouldn't need remote access.